src/explanation/skater_wrapper.py

```diff
@@ -42,8 +39,17 @@ def explain(skater_exp: Explanation, training_df, test_df, explanation_target):
                    label_fontsize=12,
                    ticks_fontsize=8,
                    fontname="Arial")
-    viz.save("skater_plot.svg")
-    f = open("skater_plot.svg", "r")
+    name = create_unique_name("skater_plot.svg");
+    viz.save(name)
+    if os.path.getsize(name) > 15000000:
+        return 'The file size is too big';
+    f = open(name, "r")
     response = f.read()
-    os.remove("skater_plot.svg")
+    os.remove(name)
+    os.remove(name.split('.svg')[0])
+
     return response
+
+
+def create_unique_name(name: str) -> str:
```

Comment:
Again, there is a function demanded to do this in "src/utils/file_service.py" you should be importing that one, why are we duplicating code?